src/registry_schemas/schemas/notice_of_withdrawal.json

```diff
@@ -16,9 +16,17 @@
             "properties": {
                 "filingId": {
                     "type": "integer",
-                    "title": "IDs for the FED filings."
+                    "title": "ID for the FED filing."
                 },
-                "courtOrder": { "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order#/properties/courtOrder" }
+                "courtOrder": { "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order#/properties/courtOrder"},
+                "hasTakenEffect": {
+                    "type": "boolean",
+                    "title": "One of the terms of arrangement have taken effect"
+                },
+                "partOfPoa": {
+                    "type": "boolean",
+                    "title": "Is part of a Plan of Arrangement"
```

Comment:
Think it's better to add descriptions to indicate that these two fields are for the filing to be withdrawn (Do I understand the usage correctly?)

Reply:
As in, "FED filing is part of a Plan of Arrangement"?

I like that.